src/ContinuumAbsorption/absorption_ff_neutral_metals_molecules.jl

```diff
@@ -0,0 +1,422 @@
+- `ne`: the number density of free electrons.
+
+For a description of the kwargs, see [Continuum Absorption Kwargs](@ref).
+
+# Notes
+
+This function is defined for 100 K ≤ T ≤ 15000 K and 0.5 μ ≤ λ. It uses data from 
+[John 1975, MNRAS, 170, 5](https://ui.adsabs.harvard.edu/abs/1975MNRAS.170....5J) for λ > 10 μm
+and data from [John 1975, MNRAS, 172, 305](https://ui.adsabs.harvard.edu/abs/1975MNRAS.172..305J)
+for λ ≤ 10 μm. This is the approach recomended by the latter paper. The error on these absorption
+coefficients is probably large.
+"""
+Ominus_ff = bounds_checked_absorption(
+    (ν, T, nO_I, ne) -> _combined_john_neg_ion_absorption(ν, T, nO_I, ne,
+                                                           _short_wavelength_Ominus_ff_interp,
+                                                           _long_wavelength_ff_interps["O"]),
+    ν_bound = λ_to_ν_bound(closed_interval(5.0e-5, Inf)),
+    temp_bound = closed_interval(100.0, 15000.0)
+)
+
+const _short_wavelength_H2minus_ff_interp = let
+    # this comes from Table II of John 1975, MNRAS, 172, 305 and it holds H₂⁻ ff absorption
+    # cross sections (including the correction for stimulated emission)
+    #
+    # I manually entered this table (so it's plausible that there are transcription errors)
+
+    elements = SMatrix{7,9}(
+        [153.0  79.7    67.9   54.6   42.7    34.8    29.3    24.6    20.9;
+          47.4  19.3    16.2   13.4   10.6     8.69    7.29    6.13    5.21;
+          15.6   5.22    3.92   3.19   2.59    2.14    1.80    1.52    1.30;
+           7.02  2.13    1.46   1.09   0.894   0.750   0.638   0.542   0.463;
+           3.76  1.07    0.695  0.470  0.380   0.322   0.277   0.237   0.203;
+           2.41  0.688   0.419  0.265  0.207   0.176   0.152   0.130   0.112;
+           1.30  0.347   0.210  0.123  0.0893  0.0748  0.0647  0.0558  0.0482] .* 1e-26
+    )
+
+    _build_interpolator_john75b(_typical_john75b_wavelengths, _typical_john75b_T_vals, elements)
+end
+
+"""
+    H2minus_ff(ν, T, nH2_I, ne; kwargs...)
+
+Compute the H₂⁻ free-free linear absorption coefficient (in cm⁻¹).
+
+The naming scheme for free-free absorption is counter-inutitive. This actually refers to the
+reaction:  `photon + e⁻ + H₂ I -> e⁻ + H₂ I.`
+
+# Arguments
+- `ν::AbstractVector{<:Real}`: sorted frequency vector in Hz
+- `T`: temperature in K
+- `nH2_I`: the total number density of neutral H₂.
+- `ne`: the number density of free electrons.
+
+For a description of the kwargs, see [Continuum Absorption Kwargs](@ref).
+
+# Notes
+
+This function is defined for 100 K ≤ T ≤ 15000 K and 0.5 μ ≤ λ. It uses data from 
+[John 1975, MNRAS, 170, 5](https://ui.adsabs.harvard.edu/abs/1975MNRAS.170....5J) for λ > 10 μm
+and data from [John 1975, MNRAS, 172, 305](https://ui.adsabs.harvard.edu/abs/1975MNRAS.172..305J)
+for λ ≤ 10 μm. This is the approach recomended by the latter paper. The error on these absorption
+coefficients is probably large.
+"""
```

Comment:
I think this file should be restructured somehow to avoid the very similar doctrings.  Do functions for individual molecules need to exist?

Reply:
Combining these seems sensible to me. I'm inclined to leave the metals as separate functions since they all use data from different papers, but if you feel super strongly about this, I can probably be convinced

Reply:
That seems sensible to me.  The tabulated data are in slightly different formats, correct?

Reply:
Ominus_ff uses data from John 75a and John 75b. 
- The format of John75a is pretty different from the formats of everything else - to make these quantities equivalent to the other sources, we just interpolate in T and then multiply the result by λ^2 (in units of Angstroms^2). This "common format" just requires that the computed "coefficient" is multiplied by the partial electron pressure and the number-density of the relevant neutral species.
- John 75a involves interpolating over λ and T

Even though they come from different papers, from each other the Cminus_ff & Nminus_ff data share the same format (Bell & Berrington were coauthors on both of those papers). This format is very similar to John 75b except they use θ instead of T (which obviously is an important distinction if we aren't interpolating in log-space).

So, if you felt it was important that we force everything into a single function, it's doable. It just doesn't seem like an obvious improvement (I really don't think we'll ever need to add any other negative atomic ion free-free absorption calculations beyond what we already have: `Hminus_ff`, `Heminus_ff`, `Cminus_ff`, `Nminus_ff`, and `Ominus_ff`)

Reply:
I‘ll leave this to your judgement.